ci-operator/config/openshift-knative/kourier/openshift-knative-kourier-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- from: base
```

Comment:
Is something missing here? Or the context_dir will be the root directory where the Dockerfile is located?

Reply:
The latter, this is the same in the serving-operator